pkg/services/ngalert/api/authorization.go

```diff
@@ -181,10 +181,14 @@ func (api *API) authorize(method, path string) web.Handler {
 		return middleware.ReqOrgAdmin
 
 	// Grafana-only Provisioning Read Paths
-	case http.MethodGet + "/api/provisioning/policies":
+	case http.MethodGet + "/api/provisioning/policies",
+		http.MethodGet + "/api/provisioning/contact-points":
 		return middleware.ReqSignedIn
 
-	case http.MethodPost + "/api/provisioning/policies":
+	case http.MethodPost + "/api/provisioning/policies",
+		http.MethodPost + "/api/provisioning/contact-points",
+		http.MethodPut + "/api/provisioning/contact-points",
+		http.MethodDelete + "/api/provisioning/contact-points/{ID}":
```

Comment:
please create new actions for provisioning API. I think `alert.provisioning:{crud}` would be enough

Reply:
we will take care of this before it goes live, right now it's still behind a feature flag. i created a task in our epic to make sure we don't forget it 👍